--- /home/minami/vscode/gitee/mugen/testcases/cli-test/lvm2/oe_test_lvm2_pvcreate_001.sh	2023-04-12 14:35:23.865506878 +0800
+++ /home/minami/vscode/github/mugen-riscv/mugen-riscv/testcases/cli-test/lvm2/oe_test_lvm2_pvcreate_001.sh	2023-03-24 19:51:42.292430811 +0800
@@ -31,7 +31,7 @@ function run_test() {
     pvdisplay | grep "PV UUID"
     pvcreate -y /dev/${local_disk} --setphysicalvolumesize 30MB | grep "successfully created"
     CHECK_RESULT $?
-    pvs | sed -n 3p | awk {'print$4'} | grep "30.00m"
+    pvs | awk {'print$4'} | grep "30.00m"
     CHECK_RESULT $?
     pvcreate -y /dev/${local_disk} --dataalignment 10MB | grep "successfully created"
     CHECK_RESULT $?
