--- /home/minami/vscode/gitee/mugen/testcases/doc-test/admin_guide/oe_test_mysql_create_view_select_delete_db/oe_test_mysql_create_view_select_delete_db.sh	2023-04-12 14:35:25.161506839 +0800
+++ /home/minami/vscode/github/mugen-riscv/mugen-riscv/testcases/doc-test/admin_guide/oe_test_mysql_create_view_select_delete_db/oe_test_mysql_create_view_select_delete_db.sh	2023-03-30 20:17:24.986881407 +0800
@@ -22,9 +22,13 @@ function pre_test() {
     LOG_INFO "Start environment preparation."
     yum list | grep mysql.*-server
     if [ $? -eq 0 ]; then
+        rmpck=$(yum list --installed | grep -E "mysql|mariadb" | grep server | awk -F ' ' '{print $1}')
+        if [ "${rmpck}x" != "x" ] ; then
+            DNF_REMOVE 1 "${rmpck}"
+        fi
         rm -rf /var/lib/mysql/*
-	pkgs=`yum list | grep mysql.*-server | awk -F ' ' '{print $1}'`
-        DNF_INSTALL ${pkgs}
+	    pkgs=`yum list | grep mysql.*-server | awk -F ' ' '{print $1}'`
+        DNF_INSTALL "${pkgs}"
         systemctl start mysqld
     else
         mysql_flag=1
@@ -70,6 +74,13 @@ function post_test() {
     rm -rf testlog
     test -z ${mysql_flag} || clean_mysql
     DNF_REMOVE
+    if [ "${rmpck}x" != "x" ] ; then
+        installedpck=$(yum list --installed | grep -E "mysql|mariadb" | grep server | awk -F ' ' '{print $1}')
+        if [ "${installedpck}x" != "x" ]; then
+            yum remove -y ${installedpck}
+        fi
+        yum install -y ${rmpck}
+    fi
     LOG_INFO "Finish environment cleanup."
 }
 
